vcr/pe/schema/v2/schema.go

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2023 Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package v2
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"github.com/santhosh-tekuri/jsonschema"
+	"github.com/santhosh-tekuri/jsonschema/loader"
+	"io"
+)
+
+//go:embed json-schema-draft-07.json
+var jsonSchemaDraft07SchemaData []byte
+
+const presentationDefinitionSchemaURL = "http://identity.foundation/presentation-exchange/schemas/presentation-definition.json"
```

Comment:
It's how the schemas reference each other, so it's sort of a constant.

Reply:
If the internal reference points to an non-versioned file there is probably not a lot we can do that makes sense. Would it help to add somewhere that these schemas are v2.0.0 (what we used according to RFC021) or a download date in case the file at this url changes in the future?

Reply:
Added links to the Github repo I got them from